DB設計

miyakawa2169 opened the pull request:
# What
READMEにDB設計のメモを記載し、実装上の問題がないか確認していただく。
# Why
ChatSpaceの実装を進めるにあたりまず初めのDB設計をFixさせる必要があるため。

miyakawa2169 committed:
masterブランチに誤ってcommit・pushしてしまったので、改めてブランチ作成

miyakawa2169 committed:
pull requestで確認しやすいように切り戻し

miyakawa2169 committed:
pull requestで確認しやすいようにファイル復元

yamamotokeishiro commented:
よく出来ております！
https://gyazo.com/35485d983ee23d94f48bcb7310512b87

こちらが解答となりますので、確認・修正後マージしてください！
分からない単語がございましたらご質問くださいませ

miyakawa2169 committed:
一意制約のオプションを追加

miyakawa2169 committed:
記載ミスを修正

・belongs_to の指定先テーブルが複数形になっていたので単数形に変更
・accepts_nested_attributes_forの指定先テーブルがおかしかったので変更
